packages/conf-time/conf-time.1/opam

```diff
@@ -4,13 +4,16 @@ authors: ["POSIX"]
 homepage: "https://github.com/ocaml/opam-repository/"
 bug-reports: "https://github.com/ocaml/opam-repository/issues"
 license: "GPL-1.0-or-later"
-build: [["which" "time"]]
-depends: ["conf-which" {build}]
+build: ["time" "ls"]
 depexts: [
   ["time"] {os-family = "debian"}
+  ["time"] {os-family = "ubuntu"}
   ["time"] {os-distribution = "centos"}
   ["time"] {os-distribution = "fedora"}
-  ["time"] {os-distribution = "arch"}
+  ["time"] {os-distribution = "ol"}
+  ["time"] {os-family = "suse"}
+  ["time"] {os-family = "arch"}
+  # busybox time is installed by default on Alpine, GNU-time is time@testing
```

Comment:
Busybox time is nothing like gnu time. If there is a depext for alpine, it should be used IMO.

Reply:
This is also why elpi tests are disabled on alpine

Reply:
true, but the `@testing` packages are rather flacky. I'll let it as is for now, we can think about it for next time